nix/wire-server.nix

```diff
@@ -486,11 +477,12 @@ in
 
   devEnv = pkgs.buildEnv {
     name = "wire-server-dev-env";
+    ignoreCollisions = true;
```

Comment:
Why do we need this?

Reply:
because of our way of setting up the environment is broken, we just merge nativeBuildInputs and buildInputs assuming that they will be completely disjoint, which in reality isn't the case, we have just been lucky until now.

Reply:
why can't we de-dup the merged lists of nativeBuildInputs and buildInputs?

Reply:
that's what this setting does, it ignores one of them. I don't really know how to usefully dedup that does something more clever than this.